active-record/README.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Active Record
+category: Architectural  
+language: en
+tags:
+ - Data access
+---
+
+## Intent
+The active record design pattern is a way of accessing data from databases by creating an object which contains the contents of a database row.

```

Comment:
There's a bit more to it, isn't there? Like each row having Insert, Update, and Delete